### DISCUSSION TITLE
SCAM!!!

### DISCUSSION THREAD

**Discussion by never-failing** [community/community#72956] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Be careful! Unfortunately, I couldn’t find how to send reports, they write on the forum.
https://github.com/SmartMathAi/SmartMathAi-DEX-JavaScript-Triangular-Arbitrage-Bot-V5
https://github.com/NoLookyloo/DEX-Triangular-Arbitrage-Bot-V4-NoLookyloo
https://github.com/KliRalph/DEX-Profit-Triangular-Arbitrage-Bot-New-Version
https://github.com/Web3Jeef/DEX-Triangular-Arbitrage-Bot-V4-Web3Jeef
https://github.com/RobbyTheTeacher/RobbyTheTeacher-DEX-Triangular-Arbitrage-Bot-V4
https://github.com/Rick8king/DEX-JavaScript-Triangular-Arbitrage-Bot-V5-Rick8king
https://github.com/ForNFTLove/DEX-Triangular-Arbitrage-Bot-V4-ForNFTLove
https://github.com/EaNt3Y/EaNt3Y-DEX-Triangular-Arbitrage-Bot-V4

**Comment by queenofcorgis** [comment, marked as answer]
Hi @never-failing 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
